proposals/sar_backscatter.json

```diff
@@ -0,0 +1,157 @@
+            "optional": true,
+            "default": null,
+            "schema": [
+                {
+                    "type": "string",
+                    "subtype": "collection-id"
+                },
+                {
+                    "type": "null"
+                }
+            ]
+        },
+        {
+            "name": "mask",
+            "description": "If set to `true`, a data mask is added to the bands. It indicates which values are valid (1), invalid (0) or contain no-data (null).",
+            "optional": true,
+            "default": false,
+            "schema": {
+                "type": "boolean"
+            }
+        },
+        {
+            "name": "contributing_area",
+            "description": "If set to `true`, a DEM-based local contributing area band is added. The values are given in square meters.",
+            "optional": true,
+            "default": false,
+            "schema": {
+                "type": "boolean"
+            }
+        },
+        {
+            "name": "local_incidence_angle",
+            "description": "If set to `true`, a DEM-based local incidence angle band is added. The values are given in degrees.",
+            "optional": true,
+            "default": false,
+            "schema": {
+                "type": "boolean"
+            }
+        },
+        {
+            "name": "ellipsoid_incidence_angle",
+            "description": "If set to `true`, a ellipsoidal incidence angle band is added. The values are given in degrees.",
+            "optional": true,
+            "default": false,
+            "schema": {
+                "type": "boolean"
+            }
+        },
+        {
+            "name": "gamma_sigma",
+            "description": "If set to `true`, a band is added that provides the ratio of the integrated area in the Gamma projection over the integrated area in the Sigma projection (ground).",
+            "optional": true,
+            "default": false,
+            "schema": {
+                "type": "boolean"
+            }
+        },
+        {
+            "name": "date_offset",
+            "description": "If set to `true`, a band is added that provides the acquisition date for each pixel. Values are day offset to the first observation date (in UTC) as specified in the metadata.",
+            "optional": true,
+            "default": false,
+            "schema": {
+                "type": "boolean"
+            }
+        },
+        {
+            "name": "scaling",
+            "description": "By default, all values are given in logarithmic decibel (dB) scale. If set to `linear`, the values are given in linear scale. Values are converted from linear to dB using *dB = 10*log₁₀(l)* and vice-versa.",
+            "optional": true,
+            "default": "dB",
```

Comment:
Is it a good idea to use dB scaling by default? As far as I know it is mathematically wrong to apply calculations, like aggregation or averaging, directly on dB values. By using `dB` scaling by default we would promote bad practices by default.
However I'm not that familiar with SAR at the moment, so it could be common practice there to do so.

Reply:
@soxofaan, thanks for pointing this out. I had a deeper look at the CARD4L specification. And you're right. The conversion is not required. 
We can stay with linear values: 

> Note: transformation to the logarithm decibel scale is not required or desired as this step can be easily completed by the user if necessary

For more details look here: 
https://ceos.org/ard/files/PFS/NRB/v5.0/CARD4L-PFS_Normalised_Radar_Backscatter-v5.0.pdf
Point 3.1 and 3.2

We could think about offering the conversion function afterwards. But isn't required for now.

Reply:
If it's actually not desired, I'd say let's completely remover the parameter and make it linear always. Doing multiply(10, log(x, 10)) is trivial to do in openEO processes.